src/ingestors/rodeo/index.ts

```diff
@@ -88,9 +88,11 @@ export class RodeoIngestor implements MintIngestor {
       chainId,
       contractAddress: mintAddress,
       contractMethod: 'mintFromFixedPriceSale',
-      contractParams: `[${sale_terms_id}, 1, address, "${user.address}"]`,
+      contractParams: `[${sale_terms_id}, quantity, address, "${user.address}"]`,
```

Comment:
This is basically the whole ingestion changeset to enable quantity (if supported cleanly by a contract!)